gufe/transformations/transformation.py

```diff
@@ -121,6 +121,10 @@ def dump(self, file):
         file : Union[PathLike, FileLike]
             A pathlike of filelike to save this transformation to.
         """
+        warnings.warn(
```

Comment:
A nice to have here would be to add a test for these warnings.

Reply:
Done!